mautrix/bridge/commands/handler.py

```diff
@@ -213,14 +214,17 @@ class CommandHandler:
     management_only: bool
     needs_admin: bool
     needs_auth: bool
+    is_enabled_for: IsEnabledForFunc
 
     _help_text: str
     _help_args: str
     help_section: HelpSection
 
     def __init__(self, handler: CommandHandlerFunc, management_only: bool, name: str,
                  help_text: str, help_args: str, help_section: HelpSection,
-                 needs_auth: bool, needs_admin: bool, **kwargs) -> None:
+                 needs_auth: bool, needs_admin: bool,
+                 is_enabled_for: IsEnabledForFunc = lambda _: True,
```

Comment:
Shouldn't this be True or False? (implying this is the return type declaration of that lamda)

```suggestion
                 is_enabled_for: IsEnabledForFunc = lambda _: bool,
```

Reply:
That's the default value, not the type signature. This makes the parameter optional, and backwards compatible.